docs/millennium-os/index.md

```diff
@@ -1,8 +1,8 @@
 # MillenniumOS - An "Operations System" for RepRapFirmware
 
-MillenniumOS is an addon for RepRapFirmware 3.5+ which implements some high-level functionality around work-piece probing, tool changes and tool setting that aim to make Millennium Machines the simplest route for novice machinists to make their first cuts confidently and safely.
+MillenniumOS is an add-on for RepRapFirmware 3.5+ that implements some high-level functionality around work-piece probing, tool changes, and toolsetting that aim to make Millennium Machines the simplest route for novice machinists to make their first cuts confidently and safely.
```

Comment:
remove "some" 
swap "around" to "for" 
"the aim" -> "aiming"
comma before aiming